pkg/dbmate/version.go

```diff
@@ -1,4 +1,4 @@
 package dbmate
 
 // Version of dbmate
-const Version = "2.27.0"
+const Version = "2.28.0"
```

Comment:
Bumping the version number is done as part of the release process in its own separate PR (e.g., #644) and not in a feature PR.  Can you revert this change?  Thanks.